app/scripts/controllers/mmi-controller.js

```diff
@@ -641,10 +652,17 @@ export default class MMIController extends EventEmitter {
 
   async setAccountAndNetwork(origin, address, chainId) {
     await this.appStateController.getUnlockPromise(true);
+    console.log(address);
```

Comment:
To be removed

Reply:
Fixed on 088f19b7c0402e2b6d28544094f16cc47365dcd4